virttest/test_setup.py

```diff
@@ -1367,12 +1369,10 @@ def remove_driver(self, driver=None):
         if ARCH == 'ppc64le' and driver == 'mlx5_core':
             pf_devices = self.get_pf_mlx()
             logging.info("Mellanox PF devices '%s'", pf_devices)
+            for PF in pf_devices:
+                if not self.set_vf(PF):
```

Comment:
with  `pf_devices = self.get_pf_mlx()` , `pf_devices` is the interface name of the nic, such as "em1", it can not be used in `set_vf(PF)` .
In short, `get_pf_mlx()` return nic interface, `get_pf_ids`  return full device id. they are different .

Reply:
I didn't test it on ppc host and I just kept it as same as before. If so, the former code couldn't work at all. Anyway, I will update. Thanks for your review.

Reply:
"set_vf" was changed and get_pf_ids was added  in commit af59aa5797c8c9fcb569914dfeffd17c872242bd. I think you can refer the code before this commit.

Reply:
Thanks, I updated set_vf() as it was before. It should work now.